pages/02.reference/02.constellation/landing.md

```diff
@@ -16,7 +16,7 @@ Want us to call you instead? Check out our [Webhooks](/reference/webhooks) syste
 
 ## Limits
 
-A single IP address can subscribe to up to 1,000,000 events. Beyond this number you will need multiple IP Addresses to receive events.
```

Comment:
I believe the original wording here is correct. The action you're doing is "subscribing to events", and limiting that is "subscribing to *up to X* events"

Reply:
Good catch @connor4312 . I removed those changes and only kept the fix in the Session section. Thanks!